test/patch_wordpress_test.js

```diff
@@ -29,4 +30,84 @@ describe('grunt_patch_wordpress', function () {
 			done()
 		})
 	})
+
+	describe( 'map_old_to_new_file_path', function() {
+		var file_mappings = {
+			'src/wp-admin/js/color-picker.js': 'src/js/_enqueues/lib/color-picker.js'
```

Comment:
Would be good to also add tests as well for non src files (aka checkouts of core, not develop)